examples/lessons/10/app.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-var, max-statements */
+/* eslint-disable array-bracket-spacing, no-multi-spaces */
+/* global document */
+import {
+  GL, AnimationLoop, loadTextures, loadFile, addEvents,
+  resetParameters, setParameters
+} from 'luma.gl';
+
+import {
+  Matrix4
+} from 'math.gl'
+
+import {loadWorldGeometry, World} from './world';
+
+var pitch = 0;
```

Comment:
We use `let` instead of `var`. Run `npm run lint`

Reply:
Maybe nice to collect these in an object (e.g. `Player`, or `View` etc)

Reply:
Done both, my Safari(version 10.0.0) doesn't support let and const well, which made this example can't run on Safari, but works well on Chrome, will create an ticket as we discussed offline.

Reply:
Figured out a solution to make example work on Safari without using "var", need wrap all variables with objects, and pass them to functions as parameters.